src/Library/demos/Calendar/main.blp

```diff
@@ -0,0 +1,30 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Calendar";
+  description: "Keep track of current or upcoming events";
```

Comment:
that'd make sense as a description for an end user calendar app but not here

I'll let you come up with a new suggestion

Reply:
How about : an arrangement of time into days, weeks, months, and years?